### PR TITLE
[master] Fix start of file inclusion

### PR DESCRIPTION
**Pull request by ljacomet**

### CODE REVIEW

File: docs/src/docs/asciidoc/user/serializers-copiers.adoc [hunk 1]
```diff
@@ -109,7 +109,7 @@ Implement the following interface, from package `org.ehcache.spi.serialization`:
 
 [source,java,indent=0]
 ----
-include::{sourcedir32}/api/src/main/java/org/ehcache/spi/serialization/Serializer.java[lines=21..-1]
+include::{sourcedir32}/api/src/main/java/org/ehcache/spi/serialization/Serializer.java[lines=20..-1]
```
Comment (lorban) [inline docs/src/docs/asciidoc/user/serializers-copiers.adoc]: Wouldn't it be wiser to add some "asciidoc tag comments" in the Serializer.java class file and be done with updating this file forever?

Comment (ljacomet) [inline docs/src/docs/asciidoc/user/serializers-copiers.adoc]: For some reason, I do not like having these tags leak in API source files ... can other chime in? @AbfrmBlr @albinsuresh @henri-tremblay ?

Comment (chrisdennis) [inline docs/src/docs/asciidoc/user/serializers-copiers.adoc]: It's an API file... not going to change any time soon.  To me there are two better fixes:

1. Write a Java syntax aware IncludeProcessor for AsciidoctorJ
2. Get rid of the include entirely (I think it's overkill).

Comment (chrisdennis) [inline docs/src/docs/asciidoc/user/serializers-copiers.adoc]: Also, point of process... is the result of this vote binding?

Comment (ljacomet) [inline docs/src/docs/asciidoc/user/serializers-copiers.adoc]: Given the votes cast, leaving it like that and not using tags in API files.

**Pull request merged**